java/src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -302,6 +304,114 @@ void useSubgraphConfidencesToShowMutuallyExclusiveLinkedSeedlingEventArgumentOpt
+            final Resource buk = model.createResource(bukUri);
+            final Resource russia = model.createResource(russiaUri);
+            final Resource bukIsRussian = makeRelation(model, russiaOwnsBukDocumentRelationUri, system);
+            markType(model, utils.getAssertionUri(), bukIsRussian, SeedlingOntology.GeneralAffiliation_APORA, system, 1.0);
+            markAsArgument(model, bukIsRussian, SeedlingOntology.GeneralAffiliation_APORA_Affiliate, buk, system, 1.0);
+            markAsArgument(model, bukIsRussian, SeedlingOntology.GeneralAffiliation_APORA_Affiliation, russia, system, 1.0);
+            return makeHypothesis(model, utils.getHypothesisUri(), ImmutableSet.of(bukIsRussian), system);
+        }
+
+        @Test
+        void eventArgumentBasedOnPreexistingHypothesis() {
+            // Both the BUK and Russia are required for both hypothesis and event
+            final Resource buk = makeEntity(model, bukDocumentEntityUri, system);
+            markType(model, utils.getAssertionUri(), buk, SeedlingOntology.Weapon, system, 1.0);
+
+            final Resource russia = makeEntity(model, russiaDocumentEntityUri, system);
+            markType(model, utils.getAssertionUri(), russia, SeedlingOntology.GeopoliticalEntity, system, 1.0);
+
+            // Add hypothesis to another model to simulate pre-existence
+            Model hypoModel = ModelFactory.createDefaultModel().add(model);
+            final Resource bukIsRussianHypothesis = addBukIsRussianHypothesisToModel(hypoModel, bukDocumentEntityUri, russiaDocumentEntityUri);
+
+            // Create event and event arguments
+            final Resource mh17 = makeEntity(model, mh17DocumentEntityUri, system);
+            markType(model, utils.getAssertionUri(), mh17, SeedlingOntology.Vehicle, system, 1.0);
+
+            final Resource attackOnMH17 = makeEvent(model, mh17AttackDocumentEventUri, system);
+            markType(model, utils.getAssertionUri(), attackOnMH17, SeedlingOntology.Conflict_Attack,
+                    system, 1.0);
+            markAsArgument(model, attackOnMH17, SeedlingOntology.Conflict_Attack_Target,
+                    mh17, system, null);
+            markAsArgument(model, attackOnMH17, SeedlingOntology.Conflict_Attack_Instrument,
+                    buk, system, null);
+
+            // Mark attacker argument as dependent on hypothesis
+            final Resource russiaShotMH17 = markAsArgument(model, attackOnMH17, SeedlingOntology.Conflict_Attack_Attacker, russia, system, 1.0);
+            markDependsOnHypothesis(russiaShotMH17, bukIsRussianHypothesis);
+            markConfidence(model, bukIsRussianHypothesis, 0.75, system);
+
+            // As model is now not valid AIF, must validate using both model and hypoModel
+            testHypothesisModels(hypoModel, model, "event argument based on pre-existing hypothesis");
+        }
+
+        private void testHypothesisModels(Model hypoModel, Model normalModel, String testDescription) {
```

Comment:
I'm good to move this into utils if you think it'll be useful. It might change some of the functions I made 'default' back to 'private'

Reply:
I don't feel strongly about it, and I don't know for sure that it will be useful for other classes down the road.  I only suggested it since it seems like a utility, albeit a specialized one.  I'm perfectly fine leaving it where it is if you prefer.

Reply:
Updated to move validation of model with hypothesis into TestUtils. Thanks for the suggestion. I think it looks cleaner. Let me know if you have further suggestions.